modules/oe_theme_content_consultation/config/install/core.entity_view_display.node.oe_consultation.full.yml

```diff
@@ -0,0 +1,281 @@
+      weight: 21
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Target audience'
+    group_why_we_are_consulting:
+      children:
+        - oe_consultation_aim
+      parent_name: group_page_contents
+      weight: 22
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Why we are consulting'
+    group_respond_to_the_consultation:
+      children:
+        - oe_consultation_guidelines
+        - extra_field_oe_consultation_closed_status_text
+        - oe_consultation_response_button
+      parent_name: group_page_contents
+      weight: 23
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Respond to the consultation'
+    group_consultation_outcome:
+      children:
+        - oe_consultation_outcome
+        - oe_consultation_outcome_files
+      parent_name: group_page_contents
+      weight: 24
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Consultation outcome'
+    group_additional_information:
+      children:
+        - oe_consultation_additional_info
+      parent_name: group_page_contents
+      weight: 25
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Additional information'
+    group_legal_notice:
+      children:
+        - oe_consultation_legal_info
+      parent_name: group_page_contents
+      weight: 26
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: 'Legal notice'
+    group_contacts:
+      children:
+        - oe_consultation_contacts
+      parent_name: group_page_contents
+      weight: 27
+      format_type: oe_theme_helper_in_page_navigation_item
+      region: content
+      format_settings:
+        id: ''
+        classes: ''
+      label: Contact
+id: node.oe_consultation.full
+targetEntityType: node
+bundle: oe_consultation
+mode: full
+content:
+  extra_field_oe_consultation_closed_status_text:
+    weight: 4
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  extra_field_oe_consultation_status:
+    weight: 0
+    region: content
+    settings: {  }
+    third_party_settings: {  }
+  oe_consultation_additional_info:
+    weight: 2
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_aim:
+    weight: 3
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    type: text_default
+    region: content
+  oe_consultation_contacts:
```

Comment:
Shouldn't it take full width the text if no image is present? 
![image](https://user-images.githubusercontent.com/22004498/105711739-40263900-5f19-11eb-8c17-eb6970d589d7.png)

Reply:
It is default Contact view display. Out of scope. It if is wrong would be better to create another ticket because it is used in few CTs.

Reply:
Indeed, it's the same behaviour in other content types:
<img width="1096" alt="call for proposals" src="https://user-images.githubusercontent.com/22908988/105775057-d3d62480-5f6e-11eb-876d-a0e188601fe4.png">

Reply:
I created a ticket to fix this.